SU-HSDO/suhumsci
# feat(STN-391): adds styles for padded card utility

# READY FOR REVIEW

## Summary
Adds `hb-padded-cards` utility that can be added to views or the layout builder "components' area for pages with cards, including:

- vertical cards
- horizontal cards
- structured cards
- post cards (not vertical linked)
- date stacked horizontal card
- date stacked vertical card

## Steps to Test
1. Check `npm run test` passes.
2. On a page with postcards, use layout builder to add `hb-padded-cards` to the Layout builder Components area as a class.
3. On a page with a card view, add the `hb-padded-cards` class to the view under "Contextual Filters" like so:

<img width="563" alt="Screen Shot 2020-05-21 at 4 01 23 PM" src="https://user-images.githubusercontent.com/12848168/82600740-64589b80-9b7c-11ea-8743-60f1d086603c.png">
4. Make sure to check all instances of cards, including:

- [x] vertical cards
- [x]  horizontal cards
- [x] structured cards
- [x] post cards (not vertical linked)
- [x] date stacked horizontal card
- [x] date stacked vertical card

Browser Testing

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
